ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__rosa-sts-test-operator-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
```

Comment:
Seeing the pj-rehearse error, I'm wondering if maybe using a different stream will help.

We have been using the nightly steam for rosa-sts-hypershift testing.
https://github.com/openshift/release/blob/master/ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__openshift-pipelines-hypershift-4.13-lp-rosa-hypershift.yaml#L15